sos/report/plugins/ceph_mon.py

```diff
@@ -49,7 +49,7 @@ def setup(self):
             "ceph config log",
             "ceph config generate-minimal-conf",
             "ceph config-key dump",
-            "ceph mon_status",
+            "ceph tell mon.* mon_status",
```

Comment:
What does `mon.*` typically expand out to in this context? Or rather, what is the expected output of this command (and is that any different from the syntax being replaced?)

Reply:
That (`mon.*`) would send the `mon_status` to all the mon daemons. While it'd work, I realzied it could more than the *current* mon's status (aka, where the command was issued). So I modified to work out the specific mon daemons' IDs.

Similarly, I modified the other 2 daemons too as `--id` doesn't necessarily have to be in any specific order (kind of of future proofing if args are re-ordered and/or more args are inserted at the beginning of the cmdline for example). It now looks at index of `--id` and then picks the next arg.